rasa/utils/log_utils.py

```diff
@@ -0,0 +1,119 @@
+import os
+import logging
+import sys
+from typing import Any, Dict, Optional
+
+import structlog
+from structlog_sentry import SentryProcessor
+from rasa.shared.constants import ENV_LOG_LEVEL, DEFAULT_LOG_LEVEL
+from rasa.plugin import plugin_manager
+
+
+FORCE_JSON_LOGGING = os.environ.get("FORCE_JSON_LOGGING")
+
+
+def _anonymizer(
+    _: structlog.BoundLogger, __: str, event_dict: Dict[str, Any]
+) -> Dict[str, Any]:
+    """Anonymizes event dict."""
+    anonymizable_keys = [
+        "text",
+        "response_text",
+        "user_text",
+        "slot_values",
+        "parse_data",
+        "prediction_events",
+        "tracker_latest_message",
+        "prefilled_slots",
+        "message",
+        "response",
+        "slot_candidates",
+        "rasa_event",
+        "tracker_states",
+        "current_states",
+        "old_states",
+        "current_states",
+        "successes",
+        "curr_ent",
+        "next_ent",
+        "states",
+        "entity",
+    ]
+    anonymization_pipeline = plugin_manager().hook.get_anonymization_pipeline()
+
+    if anonymization_pipeline:
+        for key in anonymizable_keys:
+            if key not in event_dict:
```

Comment:
Suggestion: We can shorten this section be removing the negation and `continue` instruction.
We can do this afterwards. It is not a required thing to change.